### ISSUE TITLE
downloader stuck on anonfiles links

### ISSUE THREAD

**Issue by nocoel** [Jules-WinnfieldX/CyberDropDownloader#141]
happens when downloading a thread, once it reaches anonfiles links it gets stuck while retrying to download indefinitely, which prevents it to finish the rest of the thread
this is the displayed error in downloader.log
`2022-05-19 16:55:30,995:DEBUG:downloaders:downloaders.py:227:Cannot connect to host cdn-113.anonfiles.com:443 ssl:True [SSLCertVerificationError: (1, "[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for 'cdn-113.anonfiles.com'. (_ssl.c:997)")]
2022-05-19 16:55:30,995:DEBUG:downloaders:downloaders.py:38:Retrying https://cdn-113.anonfiles.com/f6hbQfe2ye/2246c42a-1652972556/Murasaki%20Shikibu.7z...`
it's the exact 2 lines that keep repeating over and over.

**Comment by Jules-WinnfieldX** [comment]
This isn't an error with the downloader, this is an error with the certificate for anonfiles. They have a certificate for bayfiles.com instead of anonfiles.com.

**Comment by Jules-WinnfieldX** [comment]
I could put in a check and skip the files, but I won't be turning SSL verification off.

**Comment by nocoel** [comment]
i see, i thought that was the case, cuz it completes some anonfiles but gets stuck on a particular one.
isn't there a command to tell it to ignor anonfiles for example?

**Comment by Jules-WinnfieldX** [comment]
No, there is no command to ignore specific hosts, only media choices.

**Comment by Jules-WinnfieldX** [comment]
As soon as they fix their website and have a proper certificate with the proper domain name it'll work.

**Comment by nocoel** [comment]
i see! thanks!

**Comment by nocoel** [comment]
so i did some retries, and there was few tries where the certificate error didn't happen but it did get stuck anyway.
when i checked the logged i noticed something pretty much explained in this image: https://imgur.com/a/9tEIGKP
it scrapes 2 copies of the same link (i assume it was posted twice?) but can only verify one of them
the downloader gets stuck at 12/13 and the logger doesn't register anything else after what's shown in the image

**Comment by Jules-WinnfieldX** [comment]
Is this issue still present in the latest updates?

**Comment by nocoel** [comment]
unfortunatly, yes https://imgur.com/a/67d4RK9
would there be more information in the logger? should i upload that?

**Comment by nocoel** [comment]
it's only happening in this one thread, i did check and that link was posted twice, idk if it's worth your time to fix something that seems to be rare but maybe just wait it out untill that link dies or something, for me i found a way around it and i'm able to finish the thread without getting stuck
